src/main/java/com/lovetropics/minigames/client/game/handler/spectate/SpectatingUi.java

```diff
@@ -77,6 +77,9 @@ public static void onMouseScroll(InputEvent.MouseScrollEvent event) {
 		} else {
 			session.ui.onScrollSelection(delta);
 		}
+
+		// Prevent adjusting the spectator fly speed
+		event.setCanceled(true);
```

Comment:
ohhh, hmm- I guess we need some kind of way to change movement speed when in free camera mode! maybe we should change speed when doing ctrl+scroll while in free camera mode?

Reply:
Yeah that sounds like a good idea

Reply:
I've made it shift+control as just control was taken for zoom